src/core/qgsjsonutils.cpp

```diff
@@ -119,6 +120,12 @@ QString QgsJsonExporter::exportFeature( const QgsFeature &feature, const QVarian
     if ( mIncludeAttributes )
     {
       QgsFields fields = mLayer ? mLayer->fields() : feature.fields();
+      // List of formatters through we want to pass the values
+      QStringList formattersWhiteList;
```

Comment:
make sense to move formattersWhiteList to a new method in https://python.qgis.org/api/classQgsFieldFormatterRegistry.html accepting a format parameter?
e.g. QgsApplication::fieldFormatterRegistry()->getWhiteList('json')
In this way anyone exporting something in json should use the same export workflow instead to duplicate the whitelist... just my2c

Reply:
As I mentioned in the commit message this is a temporary solution (well, the list was already there, but it was a black list with a single element)

Reply:
+1